.env.server-a

```diff
@@ -0,0 +1,7 @@
+NODE_ENV=server-a

```

Comment:
Just wanted to note that all references of `server-a` & `server-b` in both the `.env` file names, the values declared within them, and the scripts defined in `package.json` are just generic placeholders. I would name these more appropriately to match the Discord servers the bot will be used for